server/videoModel.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+
+const schema = mongoose.Schema({
+  id: Number,
+  title: String,
+  url: String,
+  rating: Number
+});
+
+const Videos = mongoose.model("videos", schema, "videos");
```

Comment:
it is convention that model names are singular and table names are plural, i.e. 
```
const Video = mongoose.model("video", schema)
```
according to the docs here https://mongoosejs.com/docs/models.html